e2e_test/contracts/GoldToken.sol

```diff
@@ -0,0 +1,1147 @@
+pragma solidity ^0.5.13;
```

Comment:
How did you merge all those contracts into one file? Is there a tool for this?

Reply:
Yes, `forge flatten`. Very useful if you want to carry the source around without having to worry about the dependencies. Especially if you have contracts with different solidity version that rely in the same deps.